common/views/components/FindUs/FindUs.tsx

```diff
@@ -26,7 +26,11 @@ const PlainLink = styled.a`
   }
 `;
 
-const FindUs: FunctionComponent = () => (
+type Props = {
+  hideAccessibility?: boolean; // In the footer accessibility has its own column
```

Comment:
Is this something that'll be removed once we move it all over?

Reply:
If so we could maybe use the toggle value for the boolean just so it doesn't get missed?